### PR TITLE
Require minimum PHPUnit 7 as we require PHP 7.1

### PR DESCRIPTION
**Pull request by carusogabriel**

### CODE REVIEW

**Comment by malarzm** [comment]
@carusogabriel seems like more changes are required

**Comment by carusogabriel** [comment]
@malarzm Yes, most of them are PHP 7 return and scalar types, working on that :construction:

**Comment by carusogabriel** [comment]
Ready to go :+1:

File: tests/Doctrine/ODM/MongoDB/Tests/Aggregation/AggregationTestTrait.php [hunk 1]
```diff
@@ -6,14 +6,15 @@
...
 use Doctrine\ODM\MongoDB\Query\Expr as QueryExpr;
 use Documents\User;
+use PHPUnit\Framework\MockObject\MockBuilder;
 
 trait AggregationTestTrait
 {
     /**
      * @param string $className
-     * @return \PHPUnit_Framework_MockObject_MockBuilder
+     * @return MockBuilder
```
Comment (malarzm) [inline tests/Doctrine/ODM/MongoDB/Tests/Aggregation/AggregationTestTrait.php]: This is redundant now I think :)

Comment (carusogabriel) [reply tests/Doctrine/ODM/MongoDB/Tests/Aggregation/AggregationTestTrait.php]: Should we drop?

Comment (alcaeus) [reply tests/Doctrine/ODM/MongoDB/Tests/Aggregation/AggregationTestTrait.php]: Yep you can drop it - phpcs will catch it in the next PR ;)

**Pull request merged**